### PR TITLE
Feature/types chunkgroup

### PR DESCRIPTION
**Pull request by TheLarkInn**
**What kind of change does this PR introduce?**
Type annotations

**Did you add tests for your changes?**
N/A

**If relevant, link to documentation update:**
N/A

**Summary**
Add's type support for ChunkGroup

**Does this PR introduce a breaking change?**
No

**Other information**
N/A

### CODE REVIEW

**Comment by TheLarkInn** [comment]
Please hold off on this PR until #7119 get's merged. I had accidentally forked this PR from the entrypoints PR for specific type support reasons. Once merged, I'll rebase master so these changes dont appear.

**Comment by TheLarkInn** [comment]
Looks like now that #7119 is merged, this should be good to review @ooflorent @sokra @mohsen1

File: lib/ChunkGroup.js [hunk 1]
```diff
@@ -57,23 +93,46 @@ class ChunkGroup {
 		}
 	}
 
+	/**
+	 * returns the name of current ChunkGroup
+	 * @returns {string} returns the ChunkGroup name
```
Comment (ooflorent) [inline lib/ChunkGroup.js]: Could be `undefined`

**Comment by TheLarkInn** [comment]
Fixed nit, merging with Approval.

**Pull request merged**